packages/material-ui/test/Form.test.tsx

```diff
@@ -197,6 +197,18 @@ describe("single fields", () => {
       .toJSON();
     expect(tree).toMatchSnapshot();
   });
+  test("hidden field", () => {
+    const schema: JSONSchema7 = {
+      type: "string",
+    };
+    const uiSchema: UiSchema = {
+      "ui:widget": "hidden",
+    };
+    const tree = renderer
+      .create(<Form schema={schema} uiSchema={uiSchema} />)
```

Comment:
Should we add tests for a hidden object and hidden array as well?

Reply:
Played around with both cases and they aren't fixed by this PR, so a test wouldn't help. But I could offer to include a fix for them too.

Reply:
Okay, let's just not add it, and keep this PR limited in scope / easier to merge.